src/Compilers/CSharp/Portable/Symbols/FieldSymbol.cs

```diff
@@ -405,6 +405,16 @@ public virtual bool IsDefaultTupleElement
         {
             get
             {
+                Debug.Assert(!(this is TupleElementFieldSymbol) && !(this is TupleErrorFieldSymbol));
```

Comment:
>!(this is TupleElementFieldSymbol) && !(this is TupleErrorFieldSymbol) [](start = 29, length = 70)

Consider `this is not (TupleElementFieldSymbol or TupleErrorFieldSymbol)`

Reply:
And below

---
In reply to: [594693244](https://github.com/dotnet/roslyn/pull/44231#discussion_r594693244) [](ancestors = 594693244)